### DISCUSSION TITLE
"EDGE_FUNCTION_INVOCATION_FAILED" is the error message that appears when calling getFirestore() in a Nuxt3 server.

### DISCUSSION THREAD

**Discussion by zabefofoon** [vercel/community#1967] [Help] [answered]
### Summary

I am trying to add an API to the server folder in Nuxt3 and use Firestore, but I am experiencing unexpected issues. I have searched for a solution, but have been unable to find one. Is there any way to resolve this issue?

**Comment by amyegan** [comment, marked as answer]
[EDGE_FUNCTION_INVOCATION_FAILED](https://vercel.com/docs/errors/application-error-codes#edge_function_invocation_failed) is typically thrown when a function runs into an unhandled error. In that case, you can try wrapping your code in a try/catch to handle the error or log the error info. Your project's [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#) will have more error details to help you locate the source of the problem. 

If you need more help, please share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so we can get a closer look at what is happening
